openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -481,15 +497,17 @@ def build_template(
                                               process if version is created
 
         """
-        template_preset = self.get_template_preset()
-
-        if template_path is None:
-            template_path = template_preset["path"]
 
-        if keep_placeholders is None:
-            keep_placeholders = template_preset["keep_placeholder"]
-        if create_first_version is None:
-            create_first_version = template_preset["create_first_version"]
+        if any(value is None for value in [template_path,
```

Comment:
Why this condition?

Reply:
To be able to run a template build without being inside a context that has it defined in settings. So a user could pick a file as template and say "build me that one".

Basically this is needed to avoid the call to `self.get_template_preset`.